courses-app/src/components/Courses/components/CourseCard/CourseCard.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import { Button, Card } from 'antd';
+import './courseCard.less'
+export default function CourseCard() {
+  return (
+    <div>
+      <Card style={{ width: '100%', border: '1px solid green', marginBottom: '20rem', marginTop: '20rem' }}>
+        <div>
+          <p className='card-title'>Java</p>
+          <p>xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx</p>
+          <p>xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx</p>
+          <p>xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx</p>
+          <p>xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx</p>
+        </div>
+        <div style={{ marginLeft: '700rem' }}>

```

Comment:
This will cause the div overflows the container, please try to use `marginRight` and a `minWidth` for the container.